test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -547,6 +548,20 @@ def func(tensor):
         tensor = common_utils.get_whitenoise(sample_rate=44100)
         self._assert_consistency(func, tensor)
 
+    def test_phase_vocoder(self):
+        def func(tensor):
+            rate = 0.5
+            hop_length = 256
+            phase_advance = torch.linspace(
```

Comment:
This test is failing on CUDA device, as `phase_advance` is not on the same device.
Can you change this helper function as the following so that `phase_advance` is moved to the test device?

```python
    def func(tensor, device: torch.device = self.device):
        ...
        def func(tensor, device: torch.device = self.device):
            rate = 0.5
            hop_length = 256
            phase_advance = torch.linspace(
		0,
                3.14 * hop_length,
		tensor.shape[-3],
		dtype=torch.float64,
            ).to(device)[..., None]
            return F.phase_vocoder(tensor, rate, phase_advance)
```